src/os_posix/os_fs.c

```diff
@@ -612,6 +612,8 @@ __posix_file_write(
               "%s: handle-write: pwrite: failed to write %" WT_SIZET_FMT
               " bytes at offset %" PRIuMAX,
               file_handle->name, chunk, (uintmax_t)offset);
+        __wt_verbose(session, WT_VERB_TEMPORARY, "pwrite: %s: offset %" PRIu64 " len %" PRIu64,
+          file_handle->name, (uint64_t)offset, (uint64_t)len);
```

Comment:
Doesn't this mean nobody else can use `WT_VERB_TEMPORARY` until this code is removed?

Reply:
I guess it does. I thought I was the only one to use it. I am happy to change it to another verbose, perhaps add a specific value like `WT_VERB_DEBUG8392` or something more general `WT_VERB_DEBUG`.

Reply:
I do occasionally use it, just speaking for myself, so I'd support `WT_VERB_DEBUG8392`.